src/interface/playerSelection/playerInfo/Talents.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import SPELLS from 'common/SPELLS/index';
+import Icon from 'common/Icon';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+
+const FALLBACK_ICON = 'inv_misc_questionmark';
+
+class Talents extends React.PureComponent {
+  static propTypes = {
+    talents: PropTypes.array.isRequired,
+  };
+
+  render() {
+    const talents = this.props.talents;
+    const rows = [15, 30, 45, 60, 75, 90, 100];
+
+    return (
+      <>
+        <h3>
+          Talents
+        </h3>
+        <div className="talent-info">
+          {talents.map((spellId, index) => (
+            <div className="talent-info-row" key={index} style={{ marginBottom: '0.8em', fontSize: '1.3em' }}>
```

Comment:
Super nit pick, but I generally prefer `key` to be the first property always rather than anarchy.

Reply:
I've put key first everywhere else, not sure why I didn't for this one!